src/Opulence/QueryBuilders/Expression/Expression.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * Opulence
+ *
+ * @link      https://www.opulencephp.com
+ * @copyright Copyright (C) 2017 David Young
+ * @license   https://github.com/opulencephp/Opulence/blob/master/LICENSE.md
+ */
+
+namespace Opulence\QueryBuilders\Expression;
```

Comment:
Would we ever have more expression classes?  If not, it might make sense to move this to namespace `Opulence\QueryBuilders`.